packages/client/hmi-client/src/components/home/tera-project-card.vue

```diff
@@ -78,6 +88,17 @@ const emit = defineEmits<{
 	(e: 'removed', projectId: Project['id']): void;
 }>();
 
+const titleRef = ref();
+const descriptionLines = computed(() => {
+	const titleHeight = titleRef.value?.clientHeight;
+	for (let i = 1; i < 3; i++) {
+		if (titleHeight <= 17 * i) {
+			return 6 + i;
+		}
+	}
+	return 6;
+});
```

Comment:
I think you should limit it to 6 anyway and have the title limited as well, we should never have long title.

Reply:
Just made an adjustment. 3 lines for the title is the max that Neil gave me. A long title would require 7 lines for the description - a one line title would require 9 lines. (6 was my previous minimum)